Week-11/redux-exercise-1/src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { createStore } from "redux";
+
```

Comment:
please use proper folder structure to manage your redux actions, reducers and store

Reply:
Solved